proxy/src/main/java/com/wavefront/agent/listeners/tracing/JaegerThriftCollectorHandler.java

```diff
@@ -185,7 +147,10 @@ public JaegerThriftCollectorHandler(String handle,
       ThriftRequest<Collector.submitBatches_args> request) {
     for (Batch batch : request.getBody(Collector.submitBatches_args.class).getBatches()) {
       try {
-        processBatch(batch);
+        processBatch(batch, null, DEFAULT_SOURCE, proxyLevelApplicationName, spanHandler,
```

Comment:
Post this change, I think this class name is deceiving i.e. "JaegerThriftCollectorHandler" since we handle thrift formatted data over tchannel and http both. Should we rename it to "JaegerTChannelCollectorHandler"  ?

Reply:
renamed it to `JaegerTChannelCollectorHandler`